examples/list.rs

```diff
@@ -202,80 +188,159 @@ fn run_app<B: Backend>(
 
     // Create a List from all list items and highlight the currently selected one
     let items = List::new(items)
-        .block(Block::default().borders(Borders::ALL).title("List"))
+        .block(inner_block)
         .highlight_style(
             Style::default()
-                .bg(Color::LightGreen)
-                .add_modifier(Modifier::BOLD),
+                .add_modifier(Modifier::BOLD)
+                .add_modifier(Modifier::REVERSED)
+                .fg(SELECTED_STYLE_FG),
         )
-        .highlight_symbol(">> ");
+        .highlight_symbol(">");
 
     // We can now render the item list
-    f.render_stateful_widget(items, item_list_area, &mut app.items.state);
+    f.render_stateful_widget(items, inner_area, &mut app.items.state);
+
+    let info = if let Some(i) = app.items.state.selected() {
+        match app.items.items[i].2 {
+            true => "✓ DONE: ".to_string() + app.items.items[i].1,
+            false => "TODO: ".to_string() + app.items.items[i].1,
+        }
+    } else {
+        "Nothing to see here...".to_string()
+    };
+
+    // We show the list item's info under the list in this paragraph
+    let outer_info_block = Block::default()
+        .borders(Borders::NONE)
+        .fg(TEXT_COLOR)
+        .bg(TODO_HEADER_BG)
+        .title("TODO Info")
+        .title_alignment(Alignment::Center);
+    let inner_info_block = Block::default()
+        .borders(Borders::NONE)
+        .bg(NORMAL_ROW_COLOR)
+        .padding(Padding::horizontal(1));
+
+    // This is a similar process to what we did for list. outer_info_area will be used for header
+    // inner_info_area will be used for the list info.
+    let outer_info_area = area[1];
+    let inner_info_area = outer_info_block.inner(outer_info_area);
 
-    // Let's do the same for the events.
-    // The event list doesn't have any state and only displays the current state of the list.
+    // We can render the header. Inner info will be rendered later
+    f.render_widget(outer_info_block, outer_info_area);
+
+    let info_paragraph = Paragraph::new(info)
+        .block(inner_info_block)
+        .fg(TEXT_COLOR)
+        .wrap(Wrap { trim: false });
+
+    // We can now render the item info
+    f.render_widget(info_paragraph, inner_info_area);
+}
+
+// Renders event list
+fn render_events_list(f: &mut Frame, app: &App, area: Rect) {
     let events: Vec<ListItem> = app
         .events
         .iter()
         .rev()
-        .map(|&(event, level)| {
-            // Colorcode the level depending on its type
-            let s = match level {
-                "CRITICAL" => Style::default().fg(Color::Red),
-                "ERROR" => Style::default().fg(Color::Magenta),
-                "WARNING" => Style::default().fg(Color::Yellow),
-                "INFO" => Style::default().fg(Color::Blue),
+        .map(|level| {
+            // sets s to the tailwind color depending on its level
+            let s = match level.as_str() {
```

Comment:
I would rather remove this comment and rename `s` to something more explanatory.

Reply:
Oh yeah, I must have missed it. Thanks!